drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/importer/enrollment_coc.rb

```diff
@@ -29,11 +29,44 @@ def self.warehouse_class
     def self.hmis_validations
       {
         HouseholdID: [
+          {
+            class: HmisCsvValidation::NonBlankValidation,
+          },
           {
```

Comment:
Maybe take this one out until we enforce it uniformly?

Reply:
This one turns out to be really important for the LSA